### PR TITLE
Docs: Adding docusaurus redirects plugin

### PR DESCRIPTION
**Pull request by tolzhabayev**
**What this PR does / why we need it**:
This PR adds a possibility to redirect pages by using [plugin-client-redirects](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-client-redirects) plugin for docusaurus.

**Special notes for your reviewer**:
Is it bad for SEO? No, [it's not](https://developers.google.com/search/docs/crawling-indexing/301-redirects#jslocation)
> Google Search interprets and executes JavaScript using the Web Rendering Service once crawling of the URL has completed.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -125,6 +125,10 @@ Once you've filed the PR:
 - If the maintainer decides to pass on your PR, they will thank you for the contribution and explain why they won't be accepting the changes. That's ok! We still really appreciate you taking the time to do it, and we don't take that lightly. 💚
```
Comment (josmperez) [inline CONTRIBUTING.md:125]: Nit: use "okay" or "OK".

**Review by josmperez — APPROVED** [review]
LGTM.

File: CONTRIBUTING.md (hunk 1 above)
Comment (tolzhabayev) [reply CONTRIBUTING.md:125]: @josmperez changed https://github.com/grafana/plugin-tools/pull/774/commits/0fb67fdaeaf4f40fac62d0094d22aeeeac2f4baa

**Review by jackw — APPROVED** [review]
Nice! 🚀

**Pull request merged**